src/pbrt/pbrt.cpp

```diff
@@ -123,6 +124,8 @@ void InitPBRT(const PBRTOptions &opt) {
 
     InitBufferCaches();
 
+    GUI::Initialize();
```

Comment:
Should probably only do this if --interactive was selected (it looks like this is breaking the tests when they run on github runners, since GL isn't installed on them and so it leads to a glfwInit() error

Reply:
Done. (I had a quick look at the tests but gave up because there were so many failures, mostly optix-headers related.)